app/models/physical_ticket.rb

```diff
@@ -0,0 +1,9 @@
+class PhysicalTicket < ActiveRecord::Base
+  belongs_to :ticket_purchase
+  has_one :ticket, through: :ticket_purchase
+  has_one :conference, through: :ticket_purchase
+  has_one :user, through: :ticket_purchase
+
+  scope :by_conference, ->(conference) { joins(:ticket_purchase).where('ticket_purchases.conference_id = ?', conference.id) }
```

Comment:
Yes I tried that, but its not working. Same SQL exception no such column conference_id in physical_tickets table. 🤔